tests/resume.test.ts

```diff
@@ -0,0 +1,94 @@
+import { BadRequestError } from 'routing-controllers';
+import { anything, instance, verify } from 'ts-mockito';
+import { MediaType } from '../types';
+import { ResumeModel } from '../models/ResumeModel';
+import { Config } from '../config';
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+import { FileFactory } from './data/FileFactory';
+import Mocks from './mocks/MockFactory';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('upload resume', () => {
+  test('authenticated user can upload resume', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    await new PortalState()
+      .createUsers(member)
+      .write();
+
+    const resume = FileFactory.pdf(Config.file.MAX_RESUME_FILE_SIZE / 2);
+    const fileLocation = 'fake location';
+
+    const storageService = Mocks.storage(fileLocation);
+    const storageSvcInstance = instance(storageService);
+    const resumeController = ControllerFactory.resume(conn, storageSvcInstance);
```

Comment:
nit: we can inline `storageSvcInstance` since we aren't using it anywhere else:
```ts
const resumeController = ControllerFactory.resume(conn, instance(storageService));
```